model/clusters_mgmt/v1/listening_method_type.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Cluster components listening method.
+enum ListeningMethod {
+	// Uses only internal traffic.
+	ListeningMethodInternal
+
+	// Uses both external and internal traffic.
+	ListeningMethodExternal
```

Comment:
These should be `Internal` and `External`, or else when using the SDK you end up with the request being sent as
```json
{
  ...
  "listening": "listening_method_external"
}
```